draft-hendrickson-privacypass-public-metadata.md

```diff
@@ -288,13 +523,33 @@ truncated key IDs in rotation.
 TODO Security
 
 
-# IANA Considerations
+# IANA Considerations {#iana}
+
+This document extends the token type registry defined in {{Section 8.2.1 of !PROTOCOL}} with two new
+entries described in the following sub-sections.
+
+## Privately Verifiable Token Type
 
-This extends the token type registry defined in {{Section 8.2.1 of !PROTOCOL}} with a new token type of value `0xDA7A`:
+The contents of this token type registry entry are as follows:
+
+* Value: 0xDA7B
+* Name: Partially Oblivious PRF, OPRF(P-384, SHA-384)
+* Token Structure: As defined in {{private-finalize}}
+* TokenChallenge Structure: As defined in {{Section 2.1 of AUTHSCHEME}}
+* Publicly Verifiable: N
+* Public Metadata: Y
+* Private Metadata: N
+* Nk: 48
+* Nid: 32
+* Notes: N/A
+
+## Publicly Verifiable Token Type
+
+The contents of this token type registry entry are as follows:
 
 * Value: 0xDA7A
```

Comment:
If we're just including a normal tokentype in `ExtendedTokenRequest`, are these ids used? Or is the client still expected to place this 0xDA7X in the TokenRequest.token_type?

Reply:
Yes, the client is still expected to use these types in Token.